map/framework.cpp

```diff
@@ -2798,9 +2798,22 @@ void Framework::DeleteFeature(FeatureID const & fid) const
 {
   // TODO(AlexZ): Use FeatureID in the editor interface.
   osm::Editor::Instance().DeleteFeature(*GetFeatureByID(fid));
+  if (m_selectedFeature == fid)
+    m_selectedFeature = FeatureID();
 }
 
 osm::NewFeatureCategories Framework::GetEditorCategories() const
 {
   return osm::Editor::Instance().GetNewFeatureCategories();
 }
+
+bool Framework::RollBackChanges(FeatureID const & fid)
+{
+  if (m_selectedFeature == fid)
```

Comment:
Оставлю

> On May 5, 2016, at 2:37 PM, mgsergio <notifications@github.com> wrote:
> 
> In map/framework.cpp <https://github.com/mapsme/omim/pull/3098#discussion_r62176439>:
> 
> >  }
> >  
> >  osm::NewFeatureCategories Framework::GetEditorCategories() const
> >  {
> >    return osm::Editor::Instance().GetNewFeatureCategories();
> >  }
> > +
> > +bool Framework::RollBackChanges(FeatureID const & fid)
> > +{
> > +  if (m_selectedFeature == fid)
> Ты не мог оставить комментарий, зачем это нужно ресетить m_selectedFeature?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/3098/files/5ac03d285b681cb1a1426b8f077c432da310516a#r62176439>